templates/rails/app/views/layouts/application.html.erb.tt

```diff
@@ -47,11 +47,5 @@
   <%%= yield %>
 
   <%%= javascript_include_tag 'application' %>
-  <%%= javascript_tag do %>
-    $(function(){
-      $(document).foundation();
-    });
-  <%% end %>
-  <%%= yield :extra_footer %>
```

Comment:
Where would the extra_footer reside? What file type would it be? Does it exist for inserting javascript or html (or both)? I'm somewhat confused by this yield statement and I think beginners might be as well.